src/network/network_gui.cpp

```diff
@@ -1628,10 +1628,11 @@ static const NWidgetPart _nested_client_list_widgets[] = {
 		EndContainer(),
 		NWidget(NWID_HORIZONTAL),
 			NWidget(WWT_MATRIX, COLOUR_GREY, WID_CL_MATRIX), SetMinimalSize(180, 0), SetResize(1, 1), SetFill(1, 1), SetMatrixDataTip(1, 0, STR_NULL), SetScrollbar(WID_CL_SCROLLBAR),
-			NWidget(NWID_VERTICAL),
-				NWidget(NWID_VSCROLLBAR, COLOUR_GREY, WID_CL_SCROLLBAR),
-				NWidget(WWT_RESIZEBOX, COLOUR_GREY),
```

Comment:
I think it's probably better to keep scroll bar and resize box in the same container. And group the matrix and your new widget, it should help to align everything.

Reply:
Oh my, why I haven't thought like that! Thanks and I'll apply.